### PR TITLE
[ADD] Add Templates

### PR DESCRIPTION
**Pull request by rimbendhaou**
Add Templates for event, activity and task

### CODE REVIEW

**Comment by joaoalf** [comment]
@rimbendhaou There's a PEP8 problem. Check https://app.codacy.com/app/joaoalf/project-addons/pullRequest?prid=2099350

File: project_event/models/task_template.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class TaskTemplate(models.Model):
+    _name = "task.template"
+    _inherit = ['project.task']
+
+    activity_template_id = fields.Many2one(
+        'activity.template',
+        string='Template Event',
+    )
+    temp_resp_id = fields.Many2one(
```
Comment (joaoalf) [inline project_event/models/task_template.py]: @rimbendhaou Why we need this?

File: project_event/models/task_template.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from odoo import fields, models
+
+
+class TaskTemplate(models.Model):
+    _name = "task.template"
+    _inherit = ['project.task']
+
+    activity_template_id = fields.Many2one(
+        'activity.template',
+        string='Template Event',
```
Comment (joaoalf) [inline project_event/models/task_template.py]: @rimbendhaou Template Event?

File: project_event/models/project_template.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# coding: utf-8 -*-
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ProjectTemplate(models.Model):
+    _name = "project.template"
+    _inherit = ['project.project']
```
Comment (joaoalf) [inline project_event/models/project_template.py:10]: @rimbendhaou I was wondering... do we really need to inherit from `projet.project`? If we are just fullfiling a couple of fields I guess we could simply not inherit the project.

File: project_event/models/task_template.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# coding: utf-8 -*-
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class TaskTemplate(models.Model):
+    _name = "task.template"
+    _inherit = ['project.task']
```
Comment (joaoalf) [inline project_event/models/task_template.py:10]: @rimbendhaou Same comment from above

**Review by eilst — APPROVED** [review]

File: project_event/models/activity_template.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        'res.partner',
+        related='event_template_id.temp_resp_id',
+        readonly=True,
+        string='Responsible',
+        store=True,
+    )
+    task_template_ids = fields.One2many(
+        'task.template',
+        'activity_template_id',
+        string='Tasks Template',
```
Comment (eilst) [inline project_event/models/activity_template.py:26]: Task Template? rather than Tasks

File: project_event/models/task_template.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        related='activity_template_id.temp_resp_id',
+        readonly=True,
+        string='Responsible',
+        store=True,
+    )
+    duration = fields.Float(
+        string='Duration',
+    )
+    start_time = fields.Float(
+        string='Start Time',
```
Comment (eilst) [inline project_event/models/task_template.py:27]: Starting Time?

File: project_event/models/activity_template.py (hunk 5 above)
Comment (eilst) [reply project_event/models/activity_template.py:26]: On the xml you wrote "Task Template"

**Pull request merged**